light-rpc/src/main.rs

```diff
@@ -0,0 +1,15 @@
+use std::sync::Arc;
+
+use solana_client::nonblocking::rpc_client::RpcClient;
+use solana_client::nonblocking::tpu_client::TpuClient;
+
+const RPC_ADDR: &str = "http://localhost:8899";
+const WEBSOCKET_ADDR: &str = "ws://localhost:8900";
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
```

Comment:
you can use the same `Result` definition that is used by other solana projects.

Reply:
ohk, removing `anyhow` and replacing it with the repo wide `Result`

Reply:
https://github.com/blockworks-foundation/solana/pull/7/commits/1e6148a765c34ffa65ca2dbeaab4cb65e7fedf5f